benchllm/evaluator.py

```diff
@@ -5,11 +5,13 @@
 from operator import attrgetter
 from pathlib import Path
 from timeit import default_timer as timer
-from typing import List
+from typing import Any, List, Optional
```

Comment:
I don't think we are using `Any` anywhere
```suggestion
from typing import List, Optional
```